src/components/navbar/Navbar.module.scss

```diff
@@ -26,7 +26,7 @@
 
 .locationContent {
   padding: $gap-xxs;
-  margin-left: 0 !important;
+  margin-left: 25px !important;
```

Comment:
I know this fixed the calendar overlapping with the right border on Kennith's screen when I was watching him make the fix, but on my screen the overlap still happens. Maybe we could get some suggestions from the team on a more robust way to fix the overlap on all screen sizes?